Update azure-stack-manage-storage-accounts.md

GeorgeH-MS opened the pull request:
The actual default value of retention period is 0 days now. Update the document to reflect the reality.

azurecla commented:
Hi __@snobu__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@snobu : Thanks for your contribution to the Azure documentation! The author, @jtuliani, has been notified to review your proposed change.

jtuliani commented:
Looks good, thanks for the fix!

jtuliani commented:
#sign-off

tysonn commented:
#sign-off

GeorgeH-MS committed:
Update azure-stack-manage-storage-accounts.md

The actual default value of retention period is 0 days now. Update the document to reflect the reality.
